.github/workflows/psalm.yml

```diff
@@ -49,6 +49,7 @@ jobs:
           git checkout composer.json
           git checkout -m ${{ github.base_ref }}
 
+          sed -i 's/Uuid&/Uuid|/' src/Symfony/Component/Uid/Factory/TimeBasedUuidFactory.php
```

Comment:
Can we add a TODO to remember removing that line once Psalm 5.0 is released ?

Reply:
updated